src/js/biditrie.js

```diff
@@ -19,8 +19,6 @@
     Home: https://github.com/gorhill/uBlock
 */
 
-/* globals WebAssembly vAPI */
```

Comment:
JSHint gives a warning that this is redundant.

Reply:
Need a comma after `WebAssembly`.

Reply:
Never mind, I also see no warning when removing the line.

Reply:
Sorry, you were right. It was for ESLint too: https://github.com/gorhill/uBlock/pull/3850